crates/cli/src/commands/add.rs

```diff
@@ -117,6 +122,13 @@ impl PackageManager {
             &latest_version.serialize(args.save_exact),
             args.dependency_group(),
         )?;
+        if args.dependency_group() == DependencyGroup::Peer {
```

Comment:
Can you add a comment in here also similar to "Using --save-peer will add one or more packages to peerDependencies and install them as dev dependencies."